src/backend/metal/src/window.rs

```diff
@@ -105,38 +105,31 @@ impl SurfaceInner {
     }
 
     fn dimensions(&self) -> Extent2D {
-        let size = match self.view {
+        let size: CGSize = match self.view {
+            #[cfg(not(target_os = "macos"))]
             Some(view) => unsafe {
```

Comment:
nit: `Some(view) if cfg!(not(target_os = "macos")) => unsafe {` seems slightly clearer of the intent